### ISSUE TITLE
Compiling on Windows 10

### ISSUE THREAD

**Issue by duckbill360** [deepsense-ai/roi-pooling#15]
Hi,

Thanks for sharing the code.

I'm trying to use your api in my project. But it seems that your compilation instruction can only be run on a Linux system. Since I am using tensorflow on a windows system, some errors about missing `roi_pooling.so` appeared. It would be kind of you if you can provide some instructions on how to compile them on windows 10. Thanks.

**Comment by tgrel** [comment]
You're right, the compliation process is highly dependent on it being carried out on a Linux machine. I'm not sure what support does TensorFlow provide for compiling custom layers on Windows. Unfortunately I don't have access to a windows box at the moment so I can only encourage you to try to solve the issue on your own. If you succeed please describe the compilation steps in the README.md and make a pull request for this issue so that others don't have the same problem in the future.

Leaving this issue open until we know how to compile the layer on Windows.

**Comment by TheCreativeMind** [comment]
Hey,

are there any news concerning this issue? I am stuck with a windows machine, due to a couple of software restrictions.

**Comment by tgrel** [comment]
Currently I have don't have the means to try to compile it on Windows. If you manage to compile the layer on Windows, please make a Pull Request with the changes so that others can also benefit from it.
